src/shared/components/Footer/Language/SelectLanguage/SelectLanguage.test.tsx

```diff
@@ -13,7 +13,7 @@ const props = {
   open: true,
   onClose,
   currentLanguage: {
-    value: Languages.EN,
+    value: ApiLanguages.EN,
```

Comment:
Since this code is about the footer language selector:
```suggestion
    value: UiLanguages.EN,
```